std/functional.d

```diff
@@ -1225,6 +1225,15 @@ alias pipe(fun...) = compose!(Reverse!(fun));
     assert(compose!(`a + 0.5`, `to!(int)(a) + 1`, foo)(1) == 2.5);
 }
 
+private template getOverloads(alias fun)
+{
+    import std.meta : AliasSeq;
+    static if (__traits(compiles, __traits(getOverloads, __traits(parent, fun), __traits(identifier, fun))))
+        alias getOverloads = __traits(getOverloads, __traits(parent, fun), __traits(identifier, fun));
```

Comment:
Need to pass `true` as an additional argument to `__traits(getOverloads)` here in order to include template overloads. As-is, the check for `anySatisfy!(isTemplate, overloads)` below will always evaluate to `false`.